src/charm.py

```diff
@@ -0,0 +1,42 @@
+"""Operator Charm for AMS."""
+#!/usr/bin/env python3
+#
+# Copyright 2024 Canonical Ltd.  All rights reserved.
+#
```

Comment:
Why do we have a different copyright header than in the test files? Lets make it consistent.